app/src/main/java/sdp/moneyrun/ui/map/MapActivity.java

```diff
@@ -59,8 +60,7 @@
 @SuppressWarnings({"CanBeFinal", "FieldCanBeLocal", "FieldMayBeFinal"})
 public class MapActivity extends TrackedMap implements OnMapReadyCallback {
     public static final double THRESHOLD_DISTANCE = 5;
-    private static final int GAME_TIME = 10000;
-    public static int COINS_TO_PLACE = 2;
+    private static final double ZOOM_FOR_FEATURES = 15.;
```

Comment:
It will be used when we will not take the coins' location at random, so I thinks it's best to leave it there until next week when we won't pick the coins' location at random.